### PR TITLE
Renamed slimes for clarity

### PR DESCRIPTION
**Pull request by Christer2222**
## What Does This PR Do

Renames some of the xenobiology slimes for extra readability. I've discovered extra colors!
Purple -> violet.
Blue -> cyan.
Dark blue -> blue.
Dark purple -> purple.
Pink -> magenta.
Light Pink -> pink.

image
NOTE: I was not able to spawn in the slimes through the debug menu (only grey and random), so these are grey slimes color edited for clarity, I have not changed a single pixel in the dmi. The names of the slimes are the same as their extract ex. cyan slimes give cyan extracts.

## Why It's Good For The Game

This increases readability, it is now much quicker to look through the slime fridge for what slime extract you want. I've always felt weird about how so many slimes share the same base color.

## Testing

Breed some slimes, confirmed they mutated as before and fixed crashes.

## Changelog
:cl:
tweak: Changed the names of some xenobiology slimes.
/:cl:

### CODE REVIEW

**Comment by Christer2222** [comment]
> Your PR has been put on hold as your GitHub account is not linked to a Paradise forums account.
> 
> Please link your account at https://www.paradisestation.org/forum/custom/link_github_account.php
> 
> _Headcoders may ignore this in specific circumstances (Minor fix PR, trusted contributor from another codebase, anything at their discretion)_

I should be linked now

**Comment by Sirryan2002** [comment]
Could you please post before and after pictures with labels? It's not immediately clear what exact changes are being made.

**Comment by Christer2222** [comment]
> Could you please post before and after pictures with labels? It's not immediately clear what exact changes are being made.

I have edited the post to feature an image, hope it properly shows the change.

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -238,14 +238,14 @@
 					step(B, pick(NORTH,SOUTH,EAST,WEST))
 
 
-//Blue
+//Blue (cyan in game)
```
[the remark this answers is not in the document]
Comment (Christer2222) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Done. Do I resolve this conversation now?

**Comment by farie82** [comment]
Sorry for getting back to you this late.
The teams have voted against this PR being merged, I don't see any change that would change the votes given the following reasons given:
- Unecessary. Leave things alone that are consistent across codebases and that people have adapted to for literally years that has little impact on gameplay.
- simple renaming of slimes for not an impactful reason that will require a wiki change for something that seems really not needed to be done

**Comment by Christer2222** [comment]
I would like to request a rereview, even if the outcome is the same, for the following reasons:
* This was made mostly to make the life of newer players better, older players have an easier time adapting to change than newer players have adapting to the nearly duplicate names as they also have to learn so many other things at the same time. There are several smaller changes I would like to make to make it easier for newer players to join. SS13 is a big and difficult game to learn, and I think we should focus on the new player experience more.
* There are plenty of jank things (for example bugs) in the game that has been around for years that players have adapted to, but I don't think we should ignore them. The slime names might be a small thing, but fixing up on said things adds polish.
* I have never heard anyone say that a goal of ours is to be 1 to 1 with other codebases, and even if that is a goal; someone has to take the first step in order for things to improve.
* If updating the wiki is a big deal holding this back, I could, and would be willing to, learn how to do that.
* And lastly a nitpick; the word "unnecessary" is in the same category as "obviously", it doesn't tell me anything and is just malicious. It could be applied to every single PR, as the game is currently *running*. It makes me think that my feedback wasn't thought out all that well, and just made to get this month old PR out of the way. I made this PR to mostly learn how to contribute, but it feels like the bar of entry is needlessly high if smaller things can never be accepted. And while it was made to learn to contribute, I wouldn't have made it if I didn't think this was worth changing.

**Comment by Henri215** [comment]
> I would like to request a rereview, even if the outcome is the same, for the following reasons:
> 
> * This was made mostly to make the life of newer players better, older players have an easier time adapting to change than newer players have adapting to the nearly duplicate names as they also have to learn so many other things at the same time. There are several smaller changes I would like to make to make it easier for newer players to join. SS13 is a big and difficult game to learn, and I think we should focus on the new player experience more.
> * There are plenty of jank things (for example bugs) in the game that has been around for years that players have adapted to, but I don't think we should ignore them. The slime names might be a small thing, but fixing up on said things adds polish.
> * I have never heard anyone say that a goal of ours is to be 1 to 1 with other codebases, and even if that is a goal; someone has to take the first step in order for things to improve.
> * If updating the wiki is a big deal holding this back, I could, and would be willing to, learn how to do that.
> * And lastly a nitpick; the word "unnecessary" is in the same category as "obviously", it doesn't tell me anything and is just malicious. It could be applied to every single PR, as the game is currently _running_. It makes me think that my feedback wasn't thought out all that well, and just made to get this month old PR out of the way. I made this PR to mostly learn how to contribute, but it feels like the bar of entry is needlessly high if smaller things can never be accepted. And while it was made to learn to contribute, I wouldn't have made it if I didn't think this was worth changing.

Would be better if you did comment this over the discord, the design team isnt going to receive a message notification from this PR.

**Comment by Christer2222** [comment]
> > I would like to request a rereview, even if the outcome is the same, for the following reasons:
> > 
> > * This was made mostly to make the life of newer players better, older players have an easier time adapting to change than newer players have adapting to the nearly duplicate names as they also have to learn so many other things at the same time. There are several smaller changes I would like to make to make it easier for newer players to join. SS13 is a big and difficult game to learn, and I think we should focus on the new player experience more.
> > * There are plenty of jank things (for example bugs) in the game that has been around for years that players have adapted to, but I don't think we should ignore them. The slime names might be a small thing, but fixing up on said things adds polish.
> > * I have never heard anyone say that a goal of ours is to be 1 to 1 with other codebases, and even if that is a goal; someone has to take the first step in order for things to improve.
> > * If updating the wiki is a big deal holding this back, I could, and would be willing to, learn how to do that.
> > * And lastly a nitpick; the word "unnecessary" is in the same category as "obviously", it doesn't tell me anything and is just malicious. It could be applied to every single PR, as the game is currently _running_. It makes me think that my feedback wasn't thought out all that well, and just made to get this month old PR out of the way. I made this PR to mostly learn how to contribute, but it feels like the bar of entry is needlessly high if smaller things can never be accepted. And while it was made to learn to contribute, I wouldn't have made it if I didn't think this was worth changing.
> 
> Would be better if you did comment this over the discord, the design team isnt going to receive a message notification from this PR.

Thank you for notifying me about this, I am guessing this is a general thing rather than being because the PR is closed? Normally a PR is looked at when a decision is about to be made?

**Pull request closed without merging**